### PR TITLE
G 65 | Send DataflowRespone to gateway

### PR DESCRIPTION
**Pull request by tw-mahendra**

### CODE REVIEW

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/In.ProjectEKA.HipService/DataFlow/Model/DataFlowRequestResponse.cs  1
- src/In.ProjectEKA.HipService/DataFlow/DataFlowController.cs  1
- src/In.ProjectEKA.HipService/Gateway/Model/GatewayDataFlowRequestResponse.cs  1
         

Clones added
============
- test/In.ProjectEKA.HipServiceTest/DataFlow/PatientDataFlowControllerTest.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5717402&bid=18709058)

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/In.ProjectEKA.HipService/DataFlow/DataFlowController.cs  1
- src/In.ProjectEKA.HipService/Gateway/Model/GatewayDataFlowRequestResponse.cs  1
- src/In.ProjectEKA.HipService/DataFlow/Model/DataFlowRequestResponse.cs  1
         

Clones added
============
- test/In.ProjectEKA.HipServiceTest/DataFlow/PatientDataFlowControllerTest.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5717402&bid=18709058)

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/In.ProjectEKA.HipService/DataFlow/Model/DataFlowRequestResponse.cs  1
- src/In.ProjectEKA.HipService/DataFlow/DataFlowController.cs  1
- src/In.ProjectEKA.HipService/Gateway/Model/GatewayDataFlowRequestResponse.cs  1
         

Clones added
============
- test/In.ProjectEKA.HipServiceTest/DataFlow/PatientDataFlowControllerTest.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5717402&bid=18709058)

File: test/In.ProjectEKA.HipServiceTest/DataFlow/PatientDataFlowControllerTest.cs [hunk 1]
```diff
@@ -48,9 +54,38 @@ private void ShouldEnqueueDataFlowRequestAndReturnAccepted()
                 It.Is<Job>(job => job.Method.Name == "HealthInformationOf" && job.Args[0] == request),
                 It.IsAny<EnqueuedState>()
             ));
-
+
```
Comment (Nexengineer) [inline test/In.ProjectEKA.HipServiceTest/DataFlow/PatientDataFlowControllerTest.cs:57]: remove this

**Comment by ganesanarun** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- src/In.ProjectEKA.HipService/DataFlow/DataFlowController.cs  1
- src/In.ProjectEKA.HipService/DataFlow/Model/DataFlowRequestResponse.cs  1
- src/In.ProjectEKA.HipService/Gateway/Model/GatewayDataFlowRequestResponse.cs  1
         

Clones added
============
- test/In.ProjectEKA.HipServiceTest/DataFlow/PatientDataFlowControllerTest.cs  2
         
```


See the complete overview on [Codacy](https://app.codacy.com/gh/ProjectEKA/hip-service/pullRequest?prid=5717402&bid=18709058)

**Pull request merged**